# This message is for storing a description of a strawberry in 3D space
float32 x
float32 y
float32 z

# Change in distance from centroid to direction extrema
# dl=left, dr=right, du=up, dd=down, db=back, df=front
float32 dl
float32 dr
float32 du
float32 dd
float32 db
float32 df

int16 class_id
